Library/Homebrew/linkage_checker.rb

```diff
@@ -63,7 +63,7 @@ def check_undeclared_deps
       formula.build.without?(dep)
     end
     declared_deps = formula.deps.reject { |dep| filter_out.call(dep) }.map(&:name)
-    runtime_deps = keg.to_formula.runtime_dependencies(read_from_tab: false)
```

Comment:
Maybe the `read_from_tab` argument can go away entirely? The only other caller would be in `tab.rb`

Reply:
Changed my mind on this: ignore!